### ISSUE TITLE
Error: Invalid function argument - var.module_version is null

### ISSUE THREAD

**Issue by underyx** [dealmore/terraform-aws-next-js#59]
Following the custom domain example, I get this error:

```
Error: Invalid function argument

  on .terraform/modules/tf_next.next_image.lambda_content/main.tf line 10, in locals:
  10:   cdn_module_version = length(var.module_version) > 0 ? "@${var.module_version}" : ""
    |----------------
    | var.module_version is null

Invalid value for "value" parameter: argument must not be null.
```

**Comment by underyx** [comment]
The error doesn't appear on 0.7.0, only on 0.7.1

**Comment by ofhouse** [comment]
Yep, it is a regression from the [`v0.7.1`](https://github.com/dealmore/terraform-aws-next-js/releases/tag/v0.7.1) release. 🙈 

Using `v0.7.0` instead should fix it until we release a new version.

**Comment by ofhouse** [comment]
Just released [`v0.7.2`](https://github.com/dealmore/terraform-aws-next-js/releases/tag/v0.7.2) which contains a hotfix for this error.

This is just a temporary fix until we move the versioning of the image optimizer into a better shape.
Follow https://github.com/dealmore/terraform-aws-next-js-image-optimization/issues/14 for more information on this.

**Comment by underyx** [comment]
Thank you! Can confirm this fixed my issue.
